internal/subscription/model.go

```diff
@@ -14,6 +14,8 @@ type Model struct {
 	common.BaseModel
 	Name                           string
 	Topic                          string
+	RetryTopic                     string
```

Comment:
Since these are now part of the subscription model it will get persisted this to consul as well. Not sure if this is needed. Alternative would be to just have `subscription.GetRetryTopicName(...)` and `subscription.GetDLQTopicName(...)` and construct the names on the fly whenever needed.

Reply:
Yes, was thinking on this, makes sense for RetryTopic since that is internal. DLQ will still be exposed so should be fine to keep